src/groovy/org/grails/plugins/tomcat/TomcatServer.groovy

```diff
@@ -207,11 +207,16 @@ abstract class TomcatServer implements EmbeddableServer {
 
 	protected getKeyToolClass() {
 		try {
-			Class.forName 'sun.security.tools.KeyTool'
-		}
-		catch (ClassNotFoundException e) {
-			// no try/catch for this one, if neither is found let it fail
-			Class.forName 'com.ibm.crypto.tools.KeyTool'
+			Class.forName 'sun.security.tools.KeyTool.Main'
```

Comment:
It should be `sun.security.tools.keytool.Main` , isn't it? (lowercase `keytool`)